site/en/docs/extensions/migrating/remote-hosted-code/index.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: 'layouts/doc-post.njk'
+title: "Dealing with Remote Hosted Code Violations"
```

Comment:
@patrickkettner There's some really good information here. As I was reviewing, I realized this is in such an excellent format for a blog post instead of a guide under migration.

I had an idea that I wanted to run by you and @sebastianbenz:

1 - We keep this as is, plus an intro explaining the purpose of this blog post.
2 - We add more robust instructions to the already existing migration guide doc on remote code https://developer.chrome.com/docs/extensions/migrating/improve-security/.
3 - We link to the blog post on the [CWS Troubleshooting guide](https://developer.chrome.com/docs/webstore/troubleshooting/)

Thoughts?

Reply:
alright - what do yall want for next steps then?